src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -374,20 +374,29 @@ def _configure_db_dw_params(arg_ctx):
     creation_arg_group = 'Creation'
 
     arg_ctx.argument('collation',
-                     arg_group=creation_arg_group)
+                     arg_group=creation_arg_group,
+                     help='The collation of the database.')
 
     arg_ctx.argument('catalog_collation',
                      arg_group=creation_arg_group,
-                     arg_type=get_enum_type(CatalogCollationType))
+                     arg_type=get_enum_type(CatalogCollationType),
+                     help='Collation of the metadata catalog.  Allowed values:'
+                     'DATABASE_DEFAULT, SQL_Latin1_General_CP1_CI_AS.')
```

Comment:
For `arg_type=get_enum_type(XXX)`, the `Allowed values` will be automatically appended to help message.
So you don't have to write by yourself since the enum values could change in the future.